### PR TITLE
RK-377 - Weblogic example revamp

### PR DESCRIPTION
**Pull request by MickaelAlliel**

### CODE REVIEW

**Comment by MickaelAlliel** [comment]
Working example from https://docs.oracle.com/cd/E12839_01/web.1111/e13758/use_cases.htm#WSGET113
Allows easy deployment from ant task, triggering the webservice from the weblogic test client (localhost:7001/ws_utc)

File: java-weblogic/README.md [hunk 1]
```diff
@@ -4,37 +4,47 @@ A sample application for debugging Java + Oracle WebLogic using Rookout.
...
 * [Known Issues](#known-issues)
 
 ## Running locally
-1. Run the Rookout ETL Agent:
-    ``` bash
-    $ docker run -p 7486:7486 -e "ROOKOUT_TOKEN=<Your-Token>" rookout/agent
-    ```
 1. Add the Rookout Java Agent as described in [Rookout Integration Explained](#rookout-integration-explained).
-1. Build the sample helloworld WebLogic application:
+1. Start your WebLogic domain `sh WEBLOGIC_HOME/user_projects/domains/DOMAIN_NAME/bin/startWebLogic.sh`
```
Comment (LiranLast) [inline java-weblogic/README.md]: What's WebLogic? How do you install it?

File: java-weblogic/README.md [hunk 2]
```diff
@@ -4,37 +4,47 @@ A sample application for debugging Java + Oracle WebLogic using Rookout.
...
     $ cd weblogic-helloworld
-    $ ant
+    $ ant -lib $WEBLOGIC_HOME/wlserver/server/lib/weblogic.jar
     ```
-    The built web application is now situated in `build/weblogic-helloworld.war`
-1. Start your WebLogic domain `sh WEBLOGIC_HOME/user_projects/domains/DOMAIN_NAME/bin/startWebLogic.sh`
-1. Deploy to WebLogic using the console (default url [http://localhost:7001/console](http://localhost:7001/console))
+    The built web application is now situated in `output/helloWorldEar/helloWorldEar.war`
+1. Test your WebService using the [WebLogic Test Client](http://localhost:7001/ws_utc/begin.do?wsdlUrl=http://localhost:7001/HelloWorldImpl/HelloWorldService?WSDL)
 1. Go to [app.rookout.com](https://app.rookout.com/) and start debugging !
```
Comment (LiranLast) [inline java-weblogic/README.md]: Where do you explain about installing the Java Agent on WebLogic?

Comment (LiranLast) [reply java-weblogic/README.md]: Where do you set the token?

File: java-weblogic/README.md [hunk 3]
```diff
@@ -49,10 +59,14 @@ We have added Rookout to the original project by:
...
         JAVA_OPTIONS="${SAVE_JAVA_OPTIONS} -javaagent=ROOK_LOCATION/rook.jar"
    ```
+   - Set your `ROOKOUT_TOKEN`:
+   ```bash
+        export ROOKOUT_TOKEN="<Your-Rookout-Token>"
+   ```
 
 ## Known Issues
 
 - Breakpoints will be showing a Warning: Source file not found. This is caused by the way WebLogic runs web applications, the rook is still working properly but will not be able to know if the file used is matching the live one.
```
Comment (LiranLast) [inline java-weblogic/README.md]: Why is this here?

**Comment by MickaelAlliel** [comment]
I rewrote the how to run the app section but there's absolutely no need to include instructions on how to download and install WebLogic. People using this example should know how to and have it installed already anyway.

**Comment by LiranLast** [comment]
@MickaelAlliel `do not merge`?
